sqlite3.go

```diff
@@ -1813,6 +1858,23 @@ func (c *SQLiteConn) SetLimit(id int, newVal int) int {
 	return int(C._sqlite3_limit(c.db, C.int(id), C.int(newVal)))
 }
 
+// FileControl invokes the xFileControl method on a given database. The dbName
+// is the name of the database. It will default to "main" if left blank. The op
+// is one of the opcodes prefixed by "SQLITE_FCNTL_". The arg argument and
+// return code are both opcode-specific. Please see the SQLite documentation.
+//
+// See: sqlite3_file_control, https://www.sqlite.org/c3ref/file_control.html
+func (c *SQLiteConn) FileControl(dbName string, op int, arg unsafe.Pointer) int {
```

Comment:
The last argument seems somewhat troubling to me. For example, for some of the ops, it needs to be a pointer to some C struct, but the caller will be hard-pressed to do that within the confines of Go. Consequently, I'm not sure we really want to expose `sqlite3_file_control` directly like this.

Taking a page from the syscall package, we could do something like `FileControlInt` or `FileControlInt`, and use a more user-friendly type for the final parameter, performing the conversion to pointer internally.

Alternatively, we can make a method specifically for getting/setting the persistent WAL mode.

Reply:
Good call. I switched it to `FileControlInt()` instead. 👍